src/branch.coffee

```diff
@@ -5,7 +5,18 @@ import {markerPropertyName} from './branch-avoid-circular-dependency'
 export default (test, consequent, alternate = (props) -> props) ->
   ret = (Component) ->
     ConsequentAsComponent = flowMax consequent, Component
+    if (
+      not ConsequentAsComponent.displayName? or
+      ConsequentAsComponent.displayName is 'ret'
```

Comment:
Was seeing `ret` as display name, presumably from `ad-hok` using `ret = ...` in places

So treat `ret` equivalently as an unspecified `displayName`

Could DRY this up